stockflux-securities-master/src/components/InputForm.js

```diff
@@ -63,26 +64,42 @@ const InputForm = ({ match }) => {
       enabled
     };
 
-    postSecurity(securityObject)
-      .then(() => {
-        setMessages(["Security was successfully created"]);
-        setFormState(stateEnum.success);
-        setSecurityState({
-          exchange: "",
-          symbol: "",
-          name: "",
-          visible: false,
-          enabled: false
+    if (match.params.securityId) {
+      updateSecurity(match.params.securityId, securityObject)
+        .then(() => {
+          setMessages(["Security was successfully updated"]);
```

Comment:
![image](https://user-images.githubusercontent.com/32611900/62204767-c7770880-b385-11e9-9998-e712b3371f06.png)

I think a single bullet point with a "Security was successfully updated" message looks a bit odd. Could we perhaps check the length of messages and if the array just contains a single element, maybe we can just display it as a paragraph then? Then if there is more than 1 element, we can display it as a list. This is just a suggestion, feel free to push back if you disagree. Also, I think we could reduce the font-size (currently 20px) to something a bot smaller, e.g. 15px or 16 px. In my personal, subjective and biased opinion, the message text size should probably be smaller than the button or input labels. Also might be worth changing the cursor type on interactive elements (e.g. regular buttons, toggle buttons, edit and delete icons) to pointer: `cursor: pointer;`.

Reply:
I've made all these changes including adding the pointer cursor to the securities table.

By the way, if you want the alert messages to be coloured green or red for success or error you need to `run npm build` as I added css variables for that. It should look like this:

![image](https://user-images.githubusercontent.com/47323296/62212957-d071d500-b399-11e9-9911-60868fa7782a.png)